core-tests/shared/src/test/scala/zio/ScheduleSpec.scala

```diff
@@ -18,13 +18,13 @@ object ScheduleSpec extends ZIOBaseSpec {
   import ZIOTag._
 
   /**
-   * Retry `once` means that we try to exec `io`, get and error,
-   * try again to exec `io`, and whatever the output is, we return that
-   * second result.
-   * The three following tests test retry when:
-   * - the first time succeeds (no retry)
-   * - the first time fails and the second succeeds (one retry, result success)
-   * - both first time and retry fail (one retry, result failure)
+   * Retry `once` means that we try to exec `io`, get and error, try again to
+   * exec `io`, and whatever the output is, we return that second result. The
+   * three following tests test retry when:
+   *   - the first time succeeds (no retry)
+   *   - the first time fails and the second succeeds (one retry, result
+   *     success)
+   *   - both first time and retry fail (one retry, result failure)
```

Comment:
Another wrinkle: If we wish to preserve line breaks, we'll need an extra newline; much like Markdown.